src/test_workflow/benchmark_test/benchmark_test_suite_execute.py

```diff
@@ -0,0 +1,91 @@
+# Copyright OpenSearch Contributors
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+
+import glob
+import json
+import logging
+import os
+import shutil
+import subprocess
+
+import pandas as pd
+
+from system.temporary_directory import TemporaryDirectory
+from test_workflow.benchmark_test.benchmark_args import BenchmarkArgs
+from test_workflow.benchmark_test.benchmark_test_base import BenchmarkTestBase
+
+
+class ExecuteTestSuite(BenchmarkTestBase):
```

Comment:
Rename this to match the file name, `BenchmarkTestSuiteExecute`. 
Once you have made `BenchmarkTestSuite` class ABC class, replace it here. 
Also add `init` method to initialize required variables like `endpoint`, `password`, etc. 
E.g.
```
def __init__(self, args: BenchmarkArgs, endpoint: str, security: bool, password: str) -> None:
        super().__init__(args)
        self.endpoint = endpoint
        self.security = security
        self.password = password
```
Do the same for `compare` subclass. You may not need to set any other variable apart from doing `super().__init__(args)` as all the parameters for compare command are available from `args`.